openapi/features-api.yaml

```diff
@@ -0,0 +1,1166 @@
+          description: Performing Arts hospitality impact
+          $ref: '#/components/schemas/StatsField'
+        phq_attendance_sports_hospitality:
+          description: Sports hospitality impact
+          $ref: '#/components/schemas/StatsField'
+        phq_attendance_community_retail:
+          description: Community Retail impact
+          $ref: '#/components/schemas/StatsField'
+        phq_attendance_concerts_retail:
+          description: Concerts Retail impact
+          $ref: '#/components/schemas/StatsField'
+        phq_attendance_conferences_retail:
+          description: Conferences Retail impact
+          $ref: '#/components/schemas/StatsField'
+        phq_attendance_expos_retail:
+          description: Expos Retail impact
+          $ref: '#/components/schemas/StatsField'
+        phq_attendance_festivals_retail:
+          description: Festivals Retail impact
+          $ref: '#/components/schemas/StatsField'
+        phq_attendance_performing_arts_retail:
+          description: Performing Arts Retail impact
+          $ref: '#/components/schemas/StatsField'
+        phq_attendance_sports_retail:
+          description: Sports Retail impact
+          $ref: '#/components/schemas/StatsField'
+        phq_rank_daylight_savings:
+          description: |-
+            Daylight Savings  
+            _Default: False_
+          type: boolean
+          title: Phq Rank Daylight Savings
+        phq_rank_health_warnings:
+          description: |-
+            Health Warnings  
+            _Default: False_
+          type: boolean
+          title: Phq Rank Health Warnings
+        phq_rank_observances:
+          description: |-
+            Observances  
+            _Default: False_
+          type: boolean
+          title: Phq Rank Observances
+        phq_rank_public_holidays:
+          description: |-
+            Public Holidays  
+            _Default: False_
+          type: boolean
+          title: Phq Rank Public Holidays
+        phq_rank_school_holidays:
+          description: |-
+            School Holidays  
+            _Default: False_
+          type: boolean
+          title: Phq Rank School Holidays
+        phq_rank_politics:
+          description: |-
+            Politics  
+            _Default: False_
+          type: boolean
+          title: Phq Rank Politics
+        phq_rank_academic_session:
+          description: |-
+            Academic - Session  
+            _Default: False_
+          type: boolean
+          title: Phq Rank Academic Session
+        phq_rank_academic_exam:
+          description: |-
+            Academic - Exam  
+            _Default: False_
+          type: boolean
+          title: Phq Rank Academic Exam
+        phq_rank_academic_holiday:
+          description: |-
+            Academic - Holiday  
+            _Default: False_
+          type: boolean
+          title: Phq Rank Academic Holiday
+        phq_impact_observances:
```

Comment:
All `phq_impact_*` features operate the same as `phq_attendance_*` in that they can either be requested as bool or `StatsField`